arangod/Metrics/ClusterMetricsFeature.cpp

```diff
@@ -239,9 +252,11 @@ bool ClusterMetricsFeature::writeData(uint64_t version,
     return false;
   }
   auto metrics = parse(std::move(raw).get());
-  if (metrics.values.empty()) {
+  bool const currEmpty = metrics.values.empty();
+  if (currEmpty && _prevEmpty) {
     return true;
   }
```

Comment:
It's other fix.
If for some reason we remove all data, we need to remove it from our leader state.
I can just remove this if, but I think if data is constantly empty it's not a good idea to reassemble it and move version to the future.